index.html

```diff
@@ -315,7 +315,7 @@ <h4>Open Authorization (OAuth)</h4>
         <p>
 If Open Authorization is utilized for authorization, version 2.0 of the
```

Comment:
```suggestion
If Open Authorization (OAuth) is used for authorization, version 2.0 of the
```

Reply:
Using @jricher's suggestion over yours @TallTed.

Reply:
OK. I don't have a RESOLVE button, or I'd click it...